templates/rss.tpl

```diff
@@ -37,7 +37,7 @@
 		{/if}
 
 		{if $context->getPrimaryLocale()}
-			<dc:language>{$context->getPrimaryLocale()|replace:'_':'-'|strip|escape:"html"}</dc:language>
+			<dc:language>{$language|escape}</dc:language>
```

Comment:
As the `$language` doesn't depend on the `$context->getPrimaryLocale()` (which I guess should be always available anyway), then the `{if}` can be removed.

Reply:
Good catch! :sweat_smile: